File: app/src/main/java/com/kakeibo/util/UtilUI.java
package com.kakeibo.util;

import android.content.Context;
import android.content.res.TypedArray;

import com.kakeibo.R;

public class UtilUI {
    public static int getToolbarHeight(Context context) {
        final TypedArray styledAttributes = context.getTheme().obtainStyledAttributes(
                new int[]{R.attr.actionBarSize});
        int toolbarHeight = (int) styledAttributes.getDimension(0, 0);
        styledAttributes.recycle();

        return toolbarHeight;
    }

    public static int getTabsHeight(Context context) {
        return (int) context.getResources().getDimension(R.dimen.tab_height);
    }
}

File: app/src/main/java/com/kakeibo/util/UtilUI.kt
package com.kakeibo.util

import android.content.Context
import com.kakeibo.R

object UtilUI {

    fun getToolbarHeight(context: Context): Int {
        val styledAttributes = context.theme.obtainStyledAttributes(intArrayOf(R.attr.actionBarSize))
        val toolbarHeight = styledAttributes.getDimension(0, 0f).toInt()
        styledAttributes.recycle()
        return toolbarHeight
    }

    fun getTabsHeight(context: Context): Int {
        return context.resources.getDimension(R.dimen.tab_height).toInt()
    }
}